network/p2p/acp118/aggregator_test.go

```diff
@@ -0,0 +1,363 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package acp118
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/network/p2p"
+	"github.com/ava-labs/avalanchego/network/p2p/p2ptest"
+	"github.com/ava-labs/avalanchego/snow/engine/common"
+	"github.com/ava-labs/avalanchego/utils/crypto/bls"
+	"github.com/ava-labs/avalanchego/utils/logging"
+	"github.com/ava-labs/avalanchego/utils/set"
+	"github.com/ava-labs/avalanchego/vms/platformvm/warp"
+)
+
+func TestSignatureAggregator_AggregateSignatures(t *testing.T) {
+	networkID := uint32(123)
+	chainID := ids.GenerateTestID()
+
+	nodeID0 := ids.GenerateTestNodeID()
+	sk0, err := bls.NewSecretKey()
+	require.NoError(t, err)
+	pk0 := bls.PublicFromSecretKey(sk0)
+	signer0 := warp.NewSigner(sk0, networkID, chainID)
+
+	nodeID1 := ids.GenerateTestNodeID()
+	sk1, err := bls.NewSecretKey()
+	require.NoError(t, err)
+	pk1 := bls.PublicFromSecretKey(sk1)
+	signer1 := warp.NewSigner(sk1, networkID, chainID)
+
+	nodeID2 := ids.GenerateTestNodeID()
+	sk2, err := bls.NewSecretKey()
+	require.NoError(t, err)
+	pk2 := bls.PublicFromSecretKey(sk2)
+	signer2 := warp.NewSigner(sk2, networkID, chainID)
+
+	tests := []struct {
+		name        string
+		peers       map[ids.NodeID]p2p.Handler
+		ctx         context.Context
+		validators  []Validator
+		quorumNum   uint64
+		quorumDen   uint64
+		wantMsg     *warp.Message
+		wantSigners []int
+		wantErr     error
+	}{
+		{
+			name: "aggregates from all validators 1/1",
+			peers: map[ids.NodeID]p2p.Handler{
+				nodeID0: NewHandler(&testVerifier{}, signer0),
+			},
+			ctx: context.Background(),
+			validators: []Validator{
+				{
+					NodeID:    nodeID0,
+					PublicKey: pk0,
+					Weight:    1,
+				},
+			},
+			wantSigners: []int{0},
+			quorumNum:   1,
+			quorumDen:   1,
+		},
+		{
+			name: "aggregates from some validators - 1/3",
```

Comment:
Reading through the rest, this is probably fine as is, since there's already a convention to these names for success/failure, could just be more explicit in the success case

Reply:
Changed naming for these test cases